### PR TITLE
Would close DNN-8479

### PR DESCRIPTION
**Pull request by iJungleboy**
This would allow the installer to include pdb files in the installation, as described in detail in https://dnntracker.atlassian.net/browse/DNN-8479

Here an excerpt:

When users install our modules and run into an error, they have trouble reporting what exactly the error is, because the stack in the events-module is not really helpful.
BUT there is a solution: if we deploy the built PDB files from visual studio together with the .DLL files, then the error messages include exact line numbers. This is very helpful in fixing issues.
PDB files have no security implication, they are like JS-map files and only help the error-system inform the user about the code-line where something happened.
Because of this, it's important that we can also include the PDB files in the installation. Unfortunately as of now, the installer blocks this extension - telling the user it's unsafe.
We should add pdb-files to the list of allowed files in the assembly installer to aid in better user error-feedback.

### CODE REVIEW

**Comment by bdukes** [comment]
The symbols packages that DNN generates don't show any errors (they use a `ResourceFile` component to place the pdb files in the `bin` folder).  Does that work for you, or does using the `Assembly` component get you some other benefit?

**Comment by bdukes** [comment]
For the record, I have no problem with this change, just wanting to better understand the different options.

**Comment by iJungleboy** [comment]
I'm now also using the resourcefile workaround. It works, but it's very hacky / dirty / indiscoverable. IMHO this should be straight-forward and not feel like you're abusing a back-door.

**Comment by ohine** [comment]
I believe the `ResourceFile` workaround is only valid for only a single package with pdb's (for uninstallation that is). It would be great to support pdb files along side their dll components. Thanks for the contribution @iJungleboy!

**Comment by iJungleboy** [comment]
Will drop this pull - and re-create as it became stale...

**Comment by iJungleboy** [comment]
Note: this looks long, but it's only 1 simple file that changes!

File: DNN Platform/Library/Services/Installer/Installers/AssemblyInstaller.cs [hunk 1]
```diff
@@ -110,7 +110,7 @@ public override string AllowableFiles
         {
             get
             {
-                return "dll";
```
Comment (ashishpd) [inline DNN Platform/Library/Services/Installer/Installers/AssemblyInstaller.cs]: One my engineer recommended removing space after comma.

**Comment by iJungleboy** [comment]
Will cancel this and send a new pull-request with just this micro-change...
https://github.com/dnnsoftware/Dnn.Platform/pull/1769

**Comment by iJungleboy** [comment]
Second micro-change in this new pull request: https://github.com/dnnsoftware/Dnn.Platform/pull/1770

**Pull request closed without merging**